kustomize/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenShift configuration with Kustomize
+
+Resource configuration and secret consumption for OpenShift projects are managed
+using [`kustomize`](https://kubectl.docs.kubernetes.io/guides/introduction/kustomize/).
+
+## Setup
+
+1. Populate `service/secrets/*`.
+
+    Sensitive values and files are located in the
+    [Office Hours Secrets](https://www.dropbox.com/sh/n1igrgdsm4rt4uf/AAAXLbZOT7tpVk8XZEQj5E0ca?dl=0)
```

Comment:
The Dropbox folder is named after ROHQ project. Should we create a new folder instead?